storage-proto/proto/confirmed_block.proto

```diff
@@ -10,6 +10,7 @@ message ConfirmedBlock {
     repeated Reward rewards = 5;
     UnixTimestamp block_time = 6;
     BlockHeight block_height = 7;
+    NumPartitions num_partitions = 8;
```

Comment:
Oh interesting, I'm surprised that `ConfirmedBlock` didn't use `Rewards` directly. So `Rewards` is just used for storing in blockstore?

Reply:
Yes, that's right. I had forgotten that also.